lib/vra/client.rb

```diff
@@ -32,6 +32,7 @@ def initialize(opts)
       @tenant       = opts[:tenant]
       @verify_ssl   = opts.fetch(:verify_ssl, true)
       @bearer_token = PasswordMasker.new(nil)
+      @page_size    = opts.fetch(:page_size, 20)
```

Comment:
It's been 20 since v1.0 of the gem and is based on the vRA documentation examples.  Since we're now allowing the user a better way to override the page size instead of monkey-patching, I moved the default from the http_get_paginated_array! method args to the instance constructor where the user can either change it when they create the client, or use the attr_accessor to do so after the fact.